src/web-server.js

```diff
@@ -58,11 +58,13 @@ app.use(auth);
 app.use(express.static(__dirname + "/public"));
 
 function getEnv(req) {
-  if (req.query.env === undefined) {
-    return "default";
-  } else {
-    return req.query.env;
-  }
+  return req.query.env;
```

Comment:
By leaving `req.query.env` as `undefined` (instead of changing it to `"default"`), we're now aware of the difference between an omitted env query param and one that was specified. When it's omitted, we use the default env. When it was specified, we do not use the default env.